### ISSUE TITLE
Unable to load some images since upgrade to spotify v12.5.1

### ISSUE THREAD

**Issue by Burgestrand** [Burgestrand/Hallon#145]
``` ruby
image = Hallon::Image.new("spotify:image:57ddf794d82f3b88e73fdc8f6e1a3b8a26d83bc2")
```

```
Exception: Encoding::UndefinedConversionError: "\xDD" from ASCII-8BIT to UTF-8
--
0: /Users/Kim/.rvm/gems/ruby-2.0.0-p0/gems/spotify-12.5.1/lib/spotify/types/utf8_string.rb:21:in `encode'
1: /Users/Kim/.rvm/gems/ruby-2.0.0-p0/gems/spotify-12.5.1/lib/spotify/types/utf8_string.rb:21:in `to_native'
2: /Users/Kim/.rvm/gems/ruby-2.0.0-p0/gems/spotify-12.5.1/lib/spotify.rb:100:in `link_create_from_string'
3: /Users/Kim/.rvm/gems/ruby-2.0.0-p0/gems/spotify-12.5.1/lib/spotify.rb:100:in `block in method_missing'
4: /Users/Kim/.rvm/rubies/ruby-2.0.0-p0/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
5: /Users/Kim/.rvm/gems/ruby-2.0.0-p0/gems/spotify-12.5.1/lib/spotify.rb:99:in `method_missing'
6: /Users/Kim/Projects/Hallon/lib/hallon/link.rb:22:in `valid?'
7: /Users/Kim/Projects/Hallon/lib/hallon/base.rb:89:in `to_pointer'
8: /Users/Kim/Projects/Hallon/lib/hallon/image.rb:40:in `initialize
```

**Comment by Burgestrand** [comment]
That said, this hack is somewhat responsible for hiding this bug: https://github.com/Burgestrand/Hallon/blob/50262b0d693a5c2f381483b143cef06048dbfeef/lib/hallon/link.rb#L18

**Comment by Burgestrand** [comment]
Should be fixed in 113d9d90807b39c570be85df240e91691b2a5305.
